x-pack/test/api_integration/apis/management/remote_clusters/remote_clusters.js

```diff
@@ -95,28 +96,42 @@ export default function ({ getService }) {
     });
 
     describe('List', () => {
-      // FLAKY: https://github.com/elastic/kibana/issues/39486
-      it.skip('should return an array of remote clusters', async () => {
+      it('should return an array of remote clusters', async () => {
         const uri = `${API_BASE_PATH}`;
 
-        const { body } = await supertest
-          .get(uri)
-          .expect(200);
+        let totalRuns = 0;
+        const maxWaitTime = 10000;
+        const timeBetweenRuns = 1000;
+        const maxNumberOfRuns = Math.ceil(maxWaitTime / timeBetweenRuns);
 
-        expect(body).to.eql([
-          {
-            name: 'test_cluster',
-            seeds: [
-              NODE_SEED
-            ],
-            isConnected: true,
-            connectedNodesCount: 1,
-            maxConnectionsPerCluster: 3,
-            initialConnectTimeout: '30s',
-            skipUnavailable: false,
-            isConfiguredByNode: false,
+        const expectClusterConnected = async () => {
```

Comment:
We also have the `retry.try()` helper that would allow this to be rewritten as:

```js
const retry = getService('retry')

// ...

const body = await retry.try(() => {
  const { body } = await supertest.get(uri);
  if ((!body.length || !body[0].isConnected) && totalRuns < maxNumberOfRuns) {
    throw new Error('cluster not connected yet');
  }
  return body
})

expect(body).to.eql(...)
```

Reply:
@spalger Do you know if we have a doc of all those helper services available for our tests?

Reply:
We don't, sorry